modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,123 @@ h|Message
+h|Reason
+h|Message
+h|Resolution
+
+|The `role` field must be set for your localnet topology.
+|`spec.network.localnet.role: Required value`
+|You must set the `role` field.
+
+|`Primary` is not a supported value for the Localnet topology.
+|`spec.network.localnet.role: Unsupported value: "Primary": supported values: "Secondary"`
+|You must set the `role` for your Localnet topology to `Secondary`-the accepted value.
+|===
+
+.Invalid `subnets` and `ipam` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a,",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`LocalnetInvalidSubnets`
+3+|One of the following messages is returned when either the `spec.network.localnet.subnets` or `spec.network.localnet.ipam` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The optional fields, `subnets` and `ipam.mode`, have to be set together.
+|`Subnets is required with ipam.mode is Enabled or unset, and forbidden otherwise`
+|You must set the `subnets` field when the `ipam.mode` is `Enabled` or when the IPAM mode is unset as it defaults to `Enabled`.
+
+|The `spec.network.localnet.subnets` must have an acceptable value when using this optional field.
+|`The ClusterUserDefinedNetwork "localnet-empty-subnets-fail" is invalid: spec.network.localnet.subnets: Invalid value: 0: spec.network.localnet.subnets in body should have at least 1 items`
+|You must set an acceptable value for `spec.network.localnet.subnets`. Acceptable values are IPv4 and IPv6 Classless Inter-Domain Routing (CIDR) ranges that do not overlap with any CIDR ranges used by {product-title}.
+
+|The `subnet` field must be set when using the optional `spec.network.localnet.excludeSubnets` field.
+|`excludeSubnets must be unset when subnets is unset`
+|You must set the `spec.network.localnet.subnets` field when using the `spec.network.localnet.excludeSubnet` field.
+
+|The CIDR range is invalid.
+|`The ClusterUserDefinedNetwork "localnet-subnets-invalid-ipv4-cidr-fail" is invalid: spec.network.localnet.subnets[0]: Invalid value: "string": CIDR is invalid`
+|You must set an acceptable CIDR range for `spec.network.localnet.subnets` field. Acceptable values are IPv4 and IPv6 CIDR ranges which are not in use or reserved by {product-title}.
+
+|You must set the `subnets` field when the `ipam.mode` is `Enabled` or when the IPAM mode is unset because the default value is `Enabled`.
+|`Subnets is required with ipam.mode is Enabled or unset, and forbidden otherwise`.
+|You must set the `spec.network.localnet.subnets` field when using the optional `spec.network.localnet.ipam.mode`.
+
+|Setting two CIDR ranges for `spec.network.localnet.subnets` field requires that one be IPv4 and the other be a IPv6.
+|`The ClusterUserDefinedNetwork "localnet-subnets-multiple-ipv4-cidrs-fail" is invalid: spec.network.localnet.subnets: Invalid value: "array": When 2 CIDRs are set, they must be from different IP families`.
```

Comment:
I think there's too much details in this entry, how about leaving the following part?
 "When 2 CIDRs are set, they must be from different IP families"